oar/core/worksheet.py

```diff
@@ -500,5 +503,75 @@ def append_missed_cve_tracker_bugs(self, cve_tracker_bugs):
         # we need to send out notification
         return len(cve_tracker_bugs) > 0
 
+    def add_jira_to_others_section(self, jira_key, max_retries = 3, delay = 2):
+        """
+        Add jira to "others" section
+
+        Find the first available cell in section, and add jira to it.
+
+        Args:
+             jira_key(str): jira key to be added
+             max_retries: optional maximum number of attempts, default value is 3
+             delay: optional delay between worksheet calls, default value is 2
+        """
+        issue_keys = self._get_issues_from_others_section()
+
+        row_idx = LABEL_ISSUES_OTHERS_ROW
+        # Find first empty cell
+        if "" in issue_keys:
+            row_idx += issue_keys.index("")
+        else:
+            row_idx += len(issue_keys)
+
+        jira_hyperlink = self._to_hyperlink(
+            util.get_jira_link(jira_key), jira_key)
+
+        for attempt in range(1, max_retries + 1):
+            try:
+                self._ws.update_acell(f"{LABEL_ISSUES_OTHERS_COLUMN}{row_idx}", jira_hyperlink)
+                logger.info(f"Jira {jira_key} was added to test report")
+                break
+            except Exception as e:
+                logger.warning(f"Adding jira {jira_key} to test report failed on attempt: {attempt}")
+                if attempt < max_retries:
+                    time.sleep(delay)
+                else:
+                    logger.error(f"Adding jira {jira_key} to test report failed after all retries: {e}")
+
+    def is_cvp_issue_reported(self):
+        """
+        Identify the presence of CVP issue with specific cvp summary in test report
+
+        Returns:
+             bool: True if CVP issue is reported
+        """
+        jm = JiraManager(self._cs)
+        for key in self._get_cvp_issues_from_others_section():
```

Comment:
@LuboTerifaj there is a risk here, if the cell values are modified by mistake, i.e. jira issue is removed. then duplicate issue will be created. I think do query from jira system is more reliable

Reply:
Yes, that makes sense. I just sent a new commit.